one-app-statics/README.md

```diff
@@ -0,0 +1,66 @@
+<h1 align="center">
+  <img src='https://github.com/americanexpress/one-app/raw/master/one-app.png' alt="One App - One Amex" width='50%'/>
+</h1>
+
+# One App Client Assets and CDN
+
+## 🚨 Important 🚨  
+
+This Package is not meant to be installed using npm, it only serves as the CDN for the static client assets for `one-app`,
```

Comment:
This package doesn't serve as the CDN, it serves as the origin for the CDN once published to npm.

Reply:
"it only serves as the origin for the CDN of the static client assets for `one-app` once published to `npm`"